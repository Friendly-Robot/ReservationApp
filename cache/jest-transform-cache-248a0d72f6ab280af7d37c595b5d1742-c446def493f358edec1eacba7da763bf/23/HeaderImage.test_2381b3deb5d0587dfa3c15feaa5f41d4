a6486dabb8d140be0fb9d38091be578e
var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

require("react-native");

var _react = _interopRequireDefault(require("react"));

var _HeaderImage = _interopRequireDefault(require("../HeaderImage"));

var _reactTestRenderer = _interopRequireDefault(require("react-test-renderer"));

describe('<HeaderImage />', function () {
  it('should be defined', function () {
    expect(_HeaderImage.default).toBeDefined();
  });
  it('renders correctly', function () {
    var tree = _reactTestRenderer.default.create(_react.default.createElement(_HeaderImage.default, null)).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkhlYWRlckltYWdlLnRlc3QuanMiXSwibmFtZXMiOlsiZGVzY3JpYmUiLCJpdCIsImV4cGVjdCIsIkhlYWRlckltYWdlIiwidG9CZURlZmluZWQiLCJ0cmVlIiwicmVuZGVyZXIiLCJjcmVhdGUiLCJ0b0pTT04iLCJ0b01hdGNoU25hcHNob3QiXSwibWFwcGluZ3MiOiI7O0FBSUE7O0FBQ0E7O0FBQ0E7O0FBR0E7O0FBRUFBLFFBQVEsQ0FBQyxpQkFBRCxFQUFvQixZQUFNO0FBQ2hDQyxFQUFBQSxFQUFFLENBQUMsbUJBQUQsRUFBc0IsWUFBTTtBQUM1QkMsSUFBQUEsTUFBTSxDQUFDQyxvQkFBRCxDQUFOLENBQW9CQyxXQUFwQjtBQUNELEdBRkMsQ0FBRjtBQUlBSCxFQUFBQSxFQUFFLENBQUMsbUJBQUQsRUFBc0IsWUFBTTtBQUM1QixRQUFNSSxJQUFJLEdBQUdDLDJCQUFTQyxNQUFULENBQWdCLDZCQUFDLG9CQUFELE9BQWhCLEVBQWlDQyxNQUFqQyxFQUFiOztBQUNBTixJQUFBQSxNQUFNLENBQUNHLElBQUQsQ0FBTixDQUFhSSxlQUFiO0FBQ0QsR0FIQyxDQUFGO0FBSUQsQ0FUTyxDQUFSIiwic291cmNlc0NvbnRlbnQiOlsiLyoqXG4gKiBAZm9ybWF0XG4gKi9cblxuaW1wb3J0ICdyZWFjdC1uYXRpdmUnO1xuaW1wb3J0IFJlYWN0IGZyb20gJ3JlYWN0JztcbmltcG9ydCBIZWFkZXJJbWFnZSBmcm9tICcuLi9IZWFkZXJJbWFnZSc7XG5cbi8vIE5vdGU6IHRlc3QgcmVuZGVyZXIgbXVzdCBiZSByZXF1aXJlZCBhZnRlciByZWFjdC1uYXRpdmUuXG5pbXBvcnQgcmVuZGVyZXIgZnJvbSAncmVhY3QtdGVzdC1yZW5kZXJlcic7XG5cbmRlc2NyaWJlKCc8SGVhZGVySW1hZ2UgLz4nLCAoKSA9PiB7XG4gIGl0KCdzaG91bGQgYmUgZGVmaW5lZCcsICgpID0+IHtcbiAgICBleHBlY3QoSGVhZGVySW1hZ2UpLnRvQmVEZWZpbmVkKCk7XG4gIH0pO1xuXG4gIGl0KCdyZW5kZXJzIGNvcnJlY3RseScsICgpID0+IHtcbiAgICBjb25zdCB0cmVlID0gcmVuZGVyZXIuY3JlYXRlKDxIZWFkZXJJbWFnZSAvPikudG9KU09OKCk7XG4gICAgZXhwZWN0KHRyZWUpLnRvTWF0Y2hTbmFwc2hvdCgpO1xuICB9KTtcbn0pXG4iXX0=