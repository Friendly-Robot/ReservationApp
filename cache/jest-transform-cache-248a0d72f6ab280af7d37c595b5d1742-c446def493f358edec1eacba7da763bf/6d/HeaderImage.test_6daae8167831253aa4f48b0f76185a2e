70c8a57fb16ad83518d3cbbde4465d29
var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

require("react-native");

var _react = _interopRequireDefault(require("react"));

var _HeaderImage = _interopRequireDefault(require("../HeaderImage"));

var _reactTestRenderer = _interopRequireDefault(require("react-test-renderer"));

it('renders correctly', function () {
  var tree = _reactTestRenderer.default.create(_react.default.createElement(_HeaderImage.default, null)).toJSON();

  expect(tree).toMatchSnapshot();
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkhlYWRlckltYWdlLnRlc3QuanMiXSwibmFtZXMiOlsiaXQiLCJ0cmVlIiwicmVuZGVyZXIiLCJjcmVhdGUiLCJ0b0pTT04iLCJleHBlY3QiLCJ0b01hdGNoU25hcHNob3QiXSwibWFwcGluZ3MiOiI7O0FBSUE7O0FBQ0E7O0FBQ0E7O0FBR0E7O0FBRUFBLEVBQUUsQ0FBQyxtQkFBRCxFQUFzQixZQUFNO0FBQzVCLE1BQU1DLElBQUksR0FBR0MsMkJBQVNDLE1BQVQsQ0FBZ0IsNkJBQUMsb0JBQUQsT0FBaEIsRUFBaUNDLE1BQWpDLEVBQWI7O0FBQ0FDLEVBQUFBLE1BQU0sQ0FBQ0osSUFBRCxDQUFOLENBQWFLLGVBQWI7QUFDRCxDQUhDLENBQUYiLCJzb3VyY2VzQ29udGVudCI6WyIvKipcbiAqIEBmb3JtYXRcbiAqL1xuXG5pbXBvcnQgJ3JlYWN0LW5hdGl2ZSc7XG5pbXBvcnQgUmVhY3QgZnJvbSAncmVhY3QnO1xuaW1wb3J0IEhlYWRlckltYWdlIGZyb20gJy4uL0hlYWRlckltYWdlJztcblxuLy8gTm90ZTogdGVzdCByZW5kZXJlciBtdXN0IGJlIHJlcXVpcmVkIGFmdGVyIHJlYWN0LW5hdGl2ZS5cbmltcG9ydCByZW5kZXJlciBmcm9tICdyZWFjdC10ZXN0LXJlbmRlcmVyJztcblxuaXQoJ3JlbmRlcnMgY29ycmVjdGx5JywgKCkgPT4ge1xuICBjb25zdCB0cmVlID0gcmVuZGVyZXIuY3JlYXRlKDxIZWFkZXJJbWFnZSAvPikudG9KU09OKCk7XG4gIGV4cGVjdCh0cmVlKS50b01hdGNoU25hcHNob3QoKTtcbn0pO1xuIl19