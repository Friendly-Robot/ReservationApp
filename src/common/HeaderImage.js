import React from 'react';
import { ImageBackground, StyleSheet, Text, View } from 'react-native';

import { HotelFrontDesk } from './img';
import theme from '../theme';

export default HeaderImage = () => (
  <ImageBackground source={HotelFrontDesk} style={styles.image} />
);

const styles = StyleSheet.create({
  image: {
    height: 150,
    width: theme.deviceWidth,
    zIndex: 2,
  },
});