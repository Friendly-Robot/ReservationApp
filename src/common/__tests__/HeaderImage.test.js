/**
 * @format
 */

import 'react-native';
import React from 'react';
import HeaderImage from '../HeaderImage';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

describe('<HeaderImage />', () => {
  it('should be defined', () => {
    expect(HeaderImage).toBeDefined();
  });

  it('renders correctly', () => {
    const tree = renderer.create(<HeaderImage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
})
