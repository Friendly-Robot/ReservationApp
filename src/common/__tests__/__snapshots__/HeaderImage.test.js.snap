// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "height": 150,
      "width": 750,
      "zIndex": 2,
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/common/img/hotel-front-desk.jpeg",
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": 150,
          "width": 750,
        },
        undefined,
      ]
    }
  />
</View>
`;
